lang/pt-BR.json

```diff
@@ -0,0 +1,22 @@
+{
+    "VISION5E.Blindsense": "Sentido Cego",
+    "VISION5E.Burrowing": "Escavação",
```

Comment:
Is this the translation for "(to) burrow" or "burrowing"? I'm look at the "Flying" status translation of the core pt-BR module, which is "Voando". This is a gerund, right? I wonder if "Escavando" would be the accurate translation here (assuming this correct the gerund of "Escavação").

Reply:
Depends. "Burrowing" is from the movement (Like Fly, Swin, etc) or more like "Digging"? If it's from the movement, it's "Escavação". But if it's from the act, I think "Escavando" fits better.